soc/riscv/riscv-privilege/starfive_jh71xx/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Name your directories for a day when a (potentially annoyingly incompatible) successor for JH7100 exists. Yes, that means busywork in the future IF the chip is fully compatible.

Reply:
I am not sure about this, to be honest. Compatibility looks like an issue given how things are going on.
But I guess we can keep the directory name the same, as even there might be compatibility issues, these SoC belong to the same series and I think it can be tackled with different configs and conditional code and stuff. 
I am not really sure creating two directories like `starfive_jh7100` and `starfive_jh7110` would be a nice idea.

Reply:
I think `starfive_jh71xx` should be ok as well - ideally any minor inconsistencies should be handled either by devicetree compatibility strings / macros or (in the worst case) some runtime checks.

Reply:
I'm hip with this. Thanx!